### PR TITLE
Add layer5 learning paths blog

### PR DESCRIPTION
**Pull request by Shubhf**
**Description**

This PR fixes #5796 and #2323 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 66c05dd09174a5ab3e10874ab66a4ce05a8eff9c at: https://66cb0e26b3f6bcf1f22f6e41--layer5.netlify.app

File: .github/ISSUE_TEMPLATE/2024-08-06-introducing-layer5-learning-paths.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Introducing Layer5 Learning Paths"
```
Comment (sudhanshutech) [inline .github/ISSUE_TEMPLATE/2024-08-06-introducing-layer5-learning-paths.md:2]: @Shubhf you still adding a blog under .github folder . why? This is a workflow directory not a blog

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
All checks failing. @Ashparshp review carefully

**Comment by Ashparshp** [comment]
@Shubhf, will u fix?

**Comment by sudhanshutech** [comment]
@Shubhf understand the codebase then you can pick the issue again. Thanks

**Pull request closed without merging**